lib/connect/api/conversations.js

```diff
@@ -53,24 +52,21 @@ class ConversationService extends BaseService {
   }
 
   /**
-   * Get messages of the id request.
+   * Get messages of the conversation.
    *
    * @param {string} id   The unique request identifier for which publish.
    *
    * @returns {object} An object with the messages.
    */
   async getMessages(id) {
-    const conversationId = await this.getConversationIdByRequest(id);
-    const url = `/conversations/${conversationId}`;
+    const url = `/conversations/${id}`;
     const options = {
```

Comment:
The "GET" http method is the default method. Please remove the constant "options" and invoke fetch with only the url argument.